bouvet-deler/databricks_lambda/databricks_lambda_case2.py

```diff
@@ -0,0 +1,76 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Using Lambda in Databricks - Case 2
+# MAGIC 
+# MAGIC This notebooks gives a slightly complex example of using lambda in Databricks. For more information see this article (add link).
+# MAGIC 
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet3
+from pyspark.sql import Row
+
+DataSet3 = spark.createDataFrame(
+    [
+        Row(location="Oslo", people=[Row(firstName="Harald", lastName="Haraldsen"), Row(firstName="Mark", lastName="Hunter")]),
+        Row(location="Stockholm", people=[Row(firstName="Ingrid", lastName="Hansen")]),
+    ]
+)
+
+# COMMAND ----------
+
+# Define DataSet4
+from pyspark.sql import Row
+
+DataSet4 = spark.createDataFrame(
+    [
+        Row(location="Copenhagen", people=[Row(firstName="Roger", nobleName="of house Henriksen"), Row(firstName="Hans", nobleName="of house Dale")]),
```

Comment:
Same for this. Did you run black on the code?

Reply:
Fixed.